pkg/cosign/common.go

```diff
@@ -36,6 +44,10 @@ func FileExists(filename string) bool {
 }
 
 func ConfirmPrompt(msg string) (bool, error) {
+	if skipConfirmation {
```

Comment:
It would appear that this now skips confirmation for destructive actions as well:

https://github.com/sigstore/cosign/blob/9ccf708d576f27f1a5e9248a7b95cddcd551a108/cmd/cosign/cli/clean.go#L53-L54

Reply:
Added a simple change to ConfirmPrompt to pass in whether or not the action is destructive.

Reply:
I have a slight preference for 2 functions, `ConfirmPrompt` and `ConfirmPromptDestructive` because bool parameters are difficult to parse when reading the caller